### ISSUE TITLE
What is the recommended way to view the training loss curves in stats.jsonl?

### ISSUE THREAD

**Issue by ebartrum** [NVlabs/stylegan2-ada-pytorch#258]
I have run some experiments and I need to check the loss curves, is there a recommended tool for this?
I did not have tensorboard installed in my environment so I do not have the tf events file, only stats.jsonl. Worst case I can rerun them w/ tensorboard, but was wondering what tools are available for the new format.
